ampcrowd/basecrowd/models.py

```diff
@@ -150,3 +171,43 @@ def add_model_rels(self):
         # responses pertain to a task
         self.add_rel(self.response_model, self.task_model, models.ForeignKey,
                      'task', 'responses')
+
+
+class TemplateResource(models.Model):
```

Comment:
It would be good if these models lived in a different file from the abstract models that should be subclassed to implement new crowds. It might make sense to restructure into a `basecrowd/models/` directory containing a `abstract_crowd_models.py` and a `task_interface_models.py` (or somesuch).